### PR TITLE
Check in user when live stream has loaded/played

### PR DESCRIPTION
**Pull request by jordanwade**
# To Test
- Api needs to be on `cfdp-890-events-overhaul`
- Find a `Test 24 Hour Live Stream` click on it. 

Once the live stream loads the user will be checked in to that event based on the group selected in Rock.
image

image
image

### CODE REVIEW

File: src/content-single/EventContentItem/components/EventMedia.js [hunk 1]
```diff
@@ -1,37 +1,87 @@
...
 
-const EventMedia = ({ title, coverImage, videos, liveStreamSource }) => (
-  <div className="mb-2 px-3">
-    <Media
-      imageUrl={get(coverImage, 'sources[0].uri', '')}
-      videoUrl={
-        !!liveStreamSource && liveStreamSource !== ''
-          ? liveStreamSource
-          : get(videos, '[0].sources[0].uri', '')
+import ADD_ATTENDANCE from '../../../mutations/addAttendance';
```
Comment (rkdavidson) [inline src/content-single/EventContentItem/components/EventMedia.js:10]: FYI in the **web-app**, the imports/babel stuff is configured such that you can define paths easily relative to the root `src` dir like this:

```suggestion
import ADD_ATTENDANCE from 'mutations/addAttendance';
```

File: src/content-single/EventContentItem/components/EventMedia.js [hunk 2]
```diff
@@ -1,37 +1,87 @@
...
+        rounded
+        showControls
+      />
+    </div>
+  );
+};
 
 EventMedia.propTypes = {
-  title: PropTypes.string,
+  contentId: PropTypes.string,
```
Comment (rkdavidson) [inline src/content-single/EventContentItem/components/EventMedia.js:77]: Something to consider — all the `content` item props are `...spread` onto `<EventMedia>` already, so I _think_ we already have access to the content's `id` prop directly and don't need to pass it with an extra/new one from [up here](https://github.com/christfellowshipchurch/web-app/pull/168/files#diff-eae13af60d0b1242389fb8941d4f4b48R45) in the parent component.

**Review by rkdavidson — COMMENTED** [review]
Some off-hand code comments before I dive into testing.
Looks nice and simple so far though!

**Comment by rkdavidson** [comment]
Hey @asleepingpanda , question about _when_ the appropriate time to mark a checkin is.

Currently, this PR performs the checkin essentially as soon as the page loads, if the event `isLive` and the event has a checkin group defined. ([That happens here](https://github.com/christfellowshipchurch/web-app/pull/168/files#diff-fd7c3d01afad23fa1fdb2c8281ef9a60R50).)

**Should we add a short, arbitrary wait time before firing the checkin?**
Hypothetically, this may inflate attendance numbers since it _could_ consider people who "bounce" from the page as having attended, even if they didn't watch more than 1-5s.

It may produce more accurate metrics if we fire the mutation after ~5-10s.

WDYT?

**Comment by jordanwade** [comment]
> Hey @asleepingpanda , question about _when_ the appropriate time to mark a checkin is.
> 
> Currently, this PR performs the checkin essentially as soon as the page loads, if the event `isLive` and the event has a checkin group defined. ([That happens here](https://github.com/christfellowshipchurch/web-app/pull/168/files#diff-fd7c3d01afad23fa1fdb2c8281ef9a60R50).)
> 
> **Should we add a short, arbitrary wait time before firing the checkin?**
> Hypothetically, this may inflate attendance numbers since it _could_ consider people who "bounce" from the page as having attended, even if they didn't watch more than 1-5s.
> 
> It may produce more accurate metrics if we fire the mutation after ~5-10s.
> 
> WDYT?

@rkdavidson I went ahead and added a 10 second timeout and addressed your other feeback, but I'll wait to get @asleepingpanda feedback before moving forward.

**Pull request closed without merging**